samples/contrib/azure-samples/kfp-azure-databricks/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='kfp-azure-databricks',
-    version='0.1.0',
+    version='0.2.0',
```

Comment:
Maybe grab version number from __init__ to keep a single source of truth?

Reply:
Done